src/Appointment/Request.js

```diff
@@ -0,0 +1,328 @@
+import React, {useState} from 'react';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import './Request.css';
+
+class Request extends React.Component {
+
+    constructor() {
+
+      
+    super();
+
+    this.state = {
+
+      input: {},
+
+      errors: {}
+
+    };
+
+     
+
+    this.handleChange = this.handleChange.bind(this);
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+
+  }
+
+    
+
+  handleChange(event) {
+
+    let input = this.state.input;
+
+    input[event.target.name] = event.target.value;
+
+  
+
+    this.setState({
+
+      input
+
+    });
+
+  }
+
+
+     
+
+  handleSubmit(event) {
+
+    event.preventDefault();
+
+  
+
+    if(this.validate()){
+
+        console.log(this.state);
+
+  
+
+        let input = {};
+
+        input["name"] = "";
+
+        input["email"] = "";
+
+        input["phone"] = "";
+
+        input["date"] = "";
+
+
+        // input["address"] = "";
+
+        this.setState({input:input});
+
+  
+
+        alert('Appointment request is submited');
```

Comment:
For now using default browser alert is fine. But to make it look better we should create an Alert Popup to make it look good. We can also leverage. This alerts from [boostrap](https://getbootstrap.com/docs/5.3/components/alerts/) . To make look pretty and more appealing.